.github/workflows/ci.yml

```diff
@@ -18,9 +18,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        postgres-version: [16, 15, 14]
         go-version:
+          - "1.22"
           - "1.21"
+        postgres-version: [16, 15, 14]
```

Comment:
Flipped these because I find the actions naming better as `(GO_VERSION, PG_VERSION)` than the other way around.